basic-javascript/tests/project-2.test.js

```diff
@@ -2,3 +2,162 @@ const assert = require('chai').assert;
 const funcs = require('../src/project-2');
 
 // whoops.. there is no test suite for this file. You'll simply just have to create one :/
+describe('Project-2 Functions', () => {
+  let standardMockArray = [];
+
+  beforeEach(() => {
+    standardMockArray = ['Chicken Nuggets', 'Coconuts'];
+  });
+
+  afterEach(() => {
+    standardMockArray = [];
+  });
+
+  describe('`getBiggest`', () => {
+    const getBiggest = funcs.getBiggest;
+    const threeBiggerThanTwo = getBiggest(3, 2);
+    it('should be a function', () => {
+      assert.typeOf(getBiggest, 'function');
+    });
+    it('should return 3', () => {
+      assert.equal(threeBiggerThanTwo, 3);
+    });
+    it('should return a number', () => {
+      assert.typeOf(threeBiggerThanTwo, 'number');
+    });
+  });
+
+  describe('`greeting`', () => {
+    const greeting = funcs.greeting;
+    const spanishGreet = greeting('Spanish');
+    it('should be a function', () => {
+      assert.typeOf(greeting, 'function');
+    });
+    it('should return `Hola!`', () => {
+      assert.equal(spanishGreet, 'Hola!');
+    });
+  });
+
+  describe('`isTenOrFive`', () => {
+    it('should be a function', () => {
+      const isTenOrFive = funcs.isTenOrFive;
+      assert.typeOf(isTenOrFive, 'function');
+    });
+  });
+
+  describe('`isInRange`', () => {
+    it('should be a function', () => {
+      const isInRange = funcs.isInRange;
+      assert.typeOf(isInRange, 'function');
+    });
+  });
+
+  describe('`isInteger`', () => {
+    it('should be a function', () => {
+      const isInteger = funcs.isInteger;
+      assert.typeOf(isInteger, 'function');
+    });
+  });
+
+  describe('`fizzBUzz`', () => {
+    const fizzBUzz = funcs.fizzBuzz;
```

Comment:
👍 I like that you're declaring your variables in the outer scope so you can re-use them in the inner functions.